RubeusCore/Include/awerere_box_collider_object.h

```diff
@@ -113,40 +113,41 @@ namespace Rubeus
 			ACollideData tryIntersect(ASphereCollider & sphere) override;
 
 			/**
-			 * @fn		inline const RML::Vector3D getLowerLeftBound() const
+			 * @fn		inline const RML::Vector2D getLowerLeftBound() const
 			 *
 			 * @brief	Returns the lower left corner of the box collider
 			 *
 			 * @return	The lower left corner's position.
 			 */
-			inline const RML::Vector3D getLowerLeftBound() const { return m_MinExtent; }
+			inline const RML::Vector2D getLowerLeftBound() const { return m_MinExtent; }
 
 			/**
-			 * @fn		inline const RML::Vector3D getUpperRightBound() const
+			 * @fn		inline const RML::Vector2D getUpperRightBound() const
 			 *
 			 * @brief	Returns the upper right corner of the box collider
 			 *
 			 * @return	The upper right corner's position.
 			 */
-			inline const RML::Vector3D getUpperRightBound() const { return m_MaxExtent; }
+			inline const RML::Vector2D getUpperRightBound() const { return m_MaxExtent; }
 
 			/**
-			 * @fn		inline const RML::Vector3D getUpperLeftBound() const
+			 * @fn		inline const RML::Vector2D getUpperLeftBound() const
 			 *
 			 * @brief	Returns the upper left corner of the box collider
 			 *
 			 * @return	The upper left corner's position.
 			 */
-			inline RML::Vector3D getUpperLeftBound() const { return m_MinExtent + RML::Vector3D(0, m_MaxExtent.y - m_MinExtent.y, 0); }
+
```

Comment:
Remove the newline here.

Comments for functions should be of the form:
```cpp
/**
			 * @fn		inline const RML::Vector2D getUpperLeftBound() const
			 *				 *
			 * @brief	Returns the upper left corner of the box collider
			 *				 *
			 * @return	The upper left corner's position.			 
*/
inline RML::Vector2D getUpperLeftBound() const { return m_MinExtent + RML::Vector2D(0, m_MaxExtent.y - m_MinExtent.y); }
```
And NOT of the form:
```cpp
/**
			 * @fn		inline const RML::Vector2D getUpperLeftBound() const
			 *
			 * @brief	Returns the upper left corner of the box collider
			 *			
			 * @return	The upper left corner's position.				 				 
*/

inline RML::Vector2D getUpperLeftBound() const { return m_MinExtent + RML::Vector2D(0, m_MaxExtent.y - m_MinExtent.y); }
```